### PR TITLE
feature: add missing functions to the checklist

### PR DESCRIPTION
**Pull request by spacewander**

### CODE REVIEW

**Review by zhuizhuhaomeng — APPROVED** [review]

File: lj-releng [hunk 1]
```diff
@@ -197,8 +197,8 @@ sub process_file {
                                    |os|print|tonumber|math|pcall|xpcall|unpack
                                    |pairs|ipairs|assert|module|package
                                    |coroutine|[gs]etfenv|next|rawget|rawset
-                                   |loadstring|dofile|loadfile
```
Comment (doujiang24) [inline lj-releng:200]: why remove the `loadfile`?

File: lj-releng [hunk 2]
```diff
@@ -197,8 +197,8 @@ sub process_file {
...
-                                   |rawlen|select|arg|bit|debug|ngx|ndk)$/x)
```
Comment (doujiang24) [inline lj-releng:201]: the `newproxy` is a new function from lua 5.3?

Comment (doujiang24) [reply lj-releng:201]: oh, lua 5.1 already have this function. I had never used it ...

File: lj-releng (hunk 1 above)
Comment (spacewander) [reply lj-releng:200]: By mistake. Solved in the new commit.

**Pull request merged**